tools/downloader/src/open_model_zoo/model_tools/converter.py

```diff
@@ -199,8 +253,25 @@ def convert(reporter, model):
     if failed_models:
         reporter.print('FAILED:')
         for failed_model_name in failed_models:
+            try:
+                telemetry.send_event('md', 'converter_failed_models', failed_model_name)
```

Comment:
This event is also too late - if a model fails, then we should report that immediately, not at the end.

Reply:
Done